doc/build-helpers/trivial-build-helpers.chapter.md

```diff
@@ -92,6 +92,155 @@ writeShellScript "evaluate-my-file.sh" ''
 ```
 ::::
 
+### `makeDesktopItem` {#trivial-builder-makeDesktopItem}
+
+Write an [XDG desktop file](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html) to the Nix store.
```

Comment:
This link points to version 1.5 of  the spec. Is there a link that is guaranteed to always go to version 1.4? It would also be important to say that `makeDesktopItem` follows version 1.4.